packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js

```diff
@@ -1,4 +1,4 @@
-import {LightningElement, api, track} from 'lwc';
+import { LightningElement, api, track } from 'lwc';
```

Comment:
prettier seems to add these spaces. All the other files don't currently implement this rule. It's pretty annoying though. We should either change the prettier config or run it on all the quantic files and be done with it :P

Reply:
I think we should set a commit hook to run prettier :) Even if I don't agree with some of the rules.